web-ui/src/main/resources/catalog/components/viewer/wmsimport/WmsImportDirective.js

```diff
@@ -493,6 +510,79 @@
         }
       };
     }]);
+
+
+  /**
+   * @ngdoc directive
+   * @name gn_wmsimport.directive:gnEsriRestCapTreeCol
+   *
+   * @description
+   * Directive to manage a collection of nested layers from
+   * the ESRI REST capabilities document. This directive works with
+   * gnEsriCapTreeElt directive.
+   */
+  module.directive('gnEsriRestCapTreeCol', [
+    '$translate',
+    function($translate) {
+
+      var label= $translate.instant('filter');
```

Comment:
This should go on the scope and be used as `{{ ::label }}` in the template (might as well use angularjs for templating :wink:  )

Reply:
also this would remove the dependency to `$translate`

Reply:
This was copied for similar code in other existing directive...